vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp

```diff
@@ -526,29 +528,79 @@ bool FrameworkElementViewManager::UpdateProperty(
       DynamicAutomationProperties::SetAccessibilityStateCollapsed(
           element, states[static_cast<int32_t>(winrt::Microsoft::ReactNative::AccessibilityStates::Collapsed)]);
     } else if (propertyName == "accessibilityValue") {
+      winrt::hstring textValue;
+      double numericValues[static_cast<int32_t>(
```

Comment:
Using magic_enum to count the values doesn't change the fact that this is still a "count - 1" magic number. I think it would make much more sense to just define a constant specifying that there are 3 numeric values - min, max, or now, rather than bring in a whole new dependency.

Reply:
I think I agree with this - what we need here isn't a "count of values", but a "count of values that represent numeric properties".

We'll need a comment here, but either:
1) hardcode the # of items (add a comment in the IDL too, pointing people to update where the count of numeric values is used
2) split the enum into one that is "enum of numeric properties" vs. "enum of non-numeric properties", and use count of the numeric one.